### PR TITLE
[PLATFORM] Responsiveness and UI Improvements (mobile and tablet views)

### PR DESCRIPTION
**Pull request by OchiengPaul442**
#### Summary of Changes (What does this PR do?)

- Enhancing the responsiveness for mobile and tablet views.
- Implementing the sidebar toggle functionality and optimizing the sidebar collapse feature for larger screen displays.
- Included link to account creation view on login screen

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change is ready to hit production in its current state

#### Screenshots (optional)
image
image
image
image
image

### CODE REVIEW

**Review by Codebmk — CHANGES_REQUESTED** [review]
There's no border bottom on the top bar for some pages. You might have removed that logic from the topbar styles @OchiengPaul442 .  Make sure to cross check your UI outputs with the figma designs. Thanks.

image

File: platform/src/common/components/TopBar/index.jsx [hunk 1]
```diff
@@ -45,16 +56,25 @@ const TopBar = ({ topbarTitle, noBorderBottom }) => {
   }, [dropdownVisible]);
 
   return (
-    <nav
-      data-cy='topbar'
-      className={`sticky top-0 z-10 bg-white w-full px-4 py-4 lg:py-0 h-[76px] lg:px-16 ${
-        !noBorderBottom && 'border-b-[1px] border-b-grey-750'
-      }`}>
```
Comment (Codebmk) [inline platform/src/common/components/TopBar/index.jsx:52]: This is the noBorderBottom logic you removed @OchiengPaul442

Comment (OchiengPaul442) [reply platform/src/common/components/TopBar/index.jsx:52]: Thanks @Codebmk will include it

**Review by Codebmk — APPROVED** [review]
Sweet. Thanks @OchiengPaul442 
image

**Pull request merged**